packages/codemods/src/codeMods/ICodeMod.ts

```diff
@@ -0,0 +1,28 @@
+import { SourceFile } from 'ts-morph';
```

Comment:
Should this be a `.types.ts` file?

Reply:
I don't think so. At least not yet. This doesn't correspond to an actual object/component so I don't think that it makes sense to have a types file. I'm open to it though.

Reply:
Often in this case we just call the file `types.ts` (no component/object prefix).

Reply:
agree with @ecraig12345 here. also we should remove `I` prefix on interfaces.